[2025-03-17T01:11:44.118+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T01:11:44.161+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T01:11:44.179+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T01:11:44.180+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-17T01:11:44.206+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_minio_connection> on 2025-03-16 00:00:00+00:00
[2025-03-17T01:11:44.228+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=213) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T01:11:44.235+0000] {standard_task_runner.py:72} INFO - Started process 215 to run task
[2025-03-17T01:11:44.229+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_minio_dag', 'check_minio_connection', 'scheduled__2025-03-16T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmp_cvs1kqu']
[2025-03-17T01:11:44.236+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask check_minio_connection
[2025-03-17T01:11:44.344+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [running]> on host 643acaa02cac
[2025-03-17T01:11:44.675+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_minio_dag' AIRFLOW_CTX_TASK_ID='check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T00:00:00+00:00'
[2025-03-17T01:11:44.679+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T01:11:45.044+0000] {logging_mixin.py:190} INFO - Conexión a MinIO exitosa!
[2025-03-17T01:11:45.044+0000] {logging_mixin.py:190} INFO - Buckets: []
[2025-03-17T01:11:45.046+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T01:11:45.066+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T01:11:45.068+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_minio_dag, task_id=check_minio_connection, run_id=scheduled__2025-03-16T00:00:00+00:00, execution_date=20250316T000000, start_date=20250317T011144, end_date=20250317T011145
[2025-03-17T01:11:45.144+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T01:11:45.188+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T01:11:45.190+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-17T01:31:18.776+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T01:31:18.802+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T01:31:18.812+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T01:31:18.813+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-17T01:31:18.831+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_minio_connection> on 2025-03-16 00:00:00+00:00
[2025-03-17T01:31:18.838+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_minio_dag', 'check_minio_connection', 'scheduled__2025-03-16T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpv94ibni5']
[2025-03-17T01:31:18.842+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask check_minio_connection
[2025-03-17T01:31:18.843+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=247) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T01:31:18.845+0000] {standard_task_runner.py:72} INFO - Started process 248 to run task
[2025-03-17T01:31:18.923+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [running]> on host 643acaa02cac
[2025-03-17T01:31:19.120+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_minio_dag' AIRFLOW_CTX_TASK_ID='check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T00:00:00+00:00'
[2025-03-17T01:31:19.123+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T01:31:19.373+0000] {logging_mixin.py:190} INFO - Conexión a MinIO exitosa!
[2025-03-17T01:31:19.374+0000] {logging_mixin.py:190} INFO - Buckets: []
[2025-03-17T01:31:19.376+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T01:31:19.406+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T01:31:19.407+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_minio_dag, task_id=check_minio_connection, run_id=scheduled__2025-03-16T00:00:00+00:00, execution_date=20250316T000000, start_date=20250317T013118, end_date=20250317T013119
[2025-03-17T01:31:19.503+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T01:31:19.561+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T01:31:19.564+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-17T02:07:22.113+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T02:07:22.148+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T02:07:22.176+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T02:07:22.177+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-17T02:07:22.222+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_minio_connection> on 2025-03-16 00:00:00+00:00
[2025-03-17T02:07:22.256+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_minio_dag', 'check_minio_connection', 'scheduled__2025-03-16T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpvy1abwt2']
[2025-03-17T02:07:22.272+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=283) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T02:07:22.282+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask check_minio_connection
[2025-03-17T02:07:22.282+0000] {standard_task_runner.py:72} INFO - Started process 291 to run task
[2025-03-17T02:07:22.528+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_minio_dag.check_minio_connection scheduled__2025-03-16T00:00:00+00:00 [running]> on host d6608b866a00
[2025-03-17T02:07:22.773+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_minio_dag' AIRFLOW_CTX_TASK_ID='check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T00:00:00+00:00'
[2025-03-17T02:07:22.775+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T02:07:23.185+0000] {logging_mixin.py:190} INFO - Conexión a MinIO exitosa!
[2025-03-17T02:07:23.185+0000] {logging_mixin.py:190} INFO - Buckets: []
[2025-03-17T02:07:23.187+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T02:07:23.203+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T02:07:23.204+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_minio_dag, task_id=check_minio_connection, run_id=scheduled__2025-03-16T00:00:00+00:00, execution_date=20250316T000000, start_date=20250317T020722, end_date=20250317T020723
[2025-03-17T02:07:23.275+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T02:07:23.330+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-17T02:07:23.332+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
